docs/source/notebooks/sampling_conjugate_step.ipynb

```diff
@@ -0,0 +1,634 @@
+{
```

Comment:
<p>This plot isn't easy to read -- can you maybe add an xlabel?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc3/pull/4199/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='29'/>

Reply:
<p>This was quite messy. I reworked it to have two subplots (with axis labels).</p>

Reply:
That's much better, thanks!